### DISCUSSION TITLE
Provide ability to have repos not in Enlgish

### DISCUSSION THREAD

**Discussion by kant2002** [community/community#37632] [Repositories] [answered]
For now, you can create repository only if you are using basic Latin alphabet.
What I would like to ask, is ability create repositories in the non-English letter, For me Cyrillic would be enough.

I experiment with providing samples of source code in the Ukrainan and Russian languages, because I believe that seeing concepts in your mother tongue language allow you faster learn underlying principles behind software development. Also that's definitely helps children to better understand more advanced languages like C#/Java . Hopefully no need to pressure yourself to learn English as part of studying programming langauges. That should help concentrate on only one things at a time. 
For example: https://github.com/kant2002/funsharp

So my appetite grows, and I want not only source code to be in Ukrainan/Russian, but and Github repo to have proper name.

**Comment by ossanna16** [comment]
Hi there @kant2002 and welcome to our community! Thank you for the feedback 🙂

**Comment by ettaboyle** [comment, marked as answer]
Hi @kant2002 - Thank you for sharing product feedback and request. You make very valid points and I appreciate you calling this out. While we can't speak to future-looking statements beyond what is shared on the [public product roadmap](https://github.com/github/roadmap), I've added both the `product feedback` and `accessibility` labels on this post so the product teams that work on these features see this. 

Thank you!

**Comment by kant2002** [comment]
as you may see, this inconsistencies across whole product. for example PR comparison branch
![image](https://user-images.githubusercontent.com/4257079/204988682-c84e3a3d-b336-4922-b636-1786cae38715.png)
As seen on picture, some parts of UI properly handle Cyrillic characters, but some not.
